src/components/room/index.tsx

```diff
@@ -123,7 +123,13 @@ function Room({id: roomId, peer, username, initiator}: IRoom) {
     };
 
     const onHangUp = () => {
+        const videoTracks = state.localStream.getVideoTracks() || [];
```

Comment:
Use `getTracks()` to get all tracks including `audio`.

```suggestion
        const tracks = state.localStream.getTracks() || [];
```